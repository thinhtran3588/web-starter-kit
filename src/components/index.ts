export * from './AdminLayout';
export * from './AdminFooter';
export * from './Footer';
export * from './Header';
export * from './Icon';
export * from './Layout';
export * from './Sidebar';
export * from './SidebarMenu';
export * from './FormHeader';
export * from './Field';
export * from './FormFilter';
export * from './FormInput';
export * from './FormSearch';
export * from './Table';
export * from './LanguageSelection';
export * from './Link';
export * from './AuthLayout';
export * from './Head';
export * from './Button';
export * from './Grid';
export * from './Typography';
