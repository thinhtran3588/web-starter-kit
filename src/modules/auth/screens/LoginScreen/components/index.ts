export * from './EmailLogin';
export * from './PhoneNoLogin';
