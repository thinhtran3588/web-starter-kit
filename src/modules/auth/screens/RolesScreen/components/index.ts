export * from './Detail';
export * from './PermissionsTable';
