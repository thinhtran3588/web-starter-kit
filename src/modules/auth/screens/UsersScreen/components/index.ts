export * from './Detail';
export * from './PermissionsTable';
export * from './ChangePassword';
