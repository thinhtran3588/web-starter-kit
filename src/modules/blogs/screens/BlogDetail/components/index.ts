export * from './Content';
export * from './SideMenu';
