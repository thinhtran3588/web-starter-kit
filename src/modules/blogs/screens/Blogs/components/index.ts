export * from './Categories';
export * from './BlogsList';
