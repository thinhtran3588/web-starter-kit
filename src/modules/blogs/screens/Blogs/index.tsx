import React from 'react';
import { Grid } from '@material-ui/core';
import { Layout } from '@app/components';
import { Categories, BlogsList } from './components';

export const Blogs = (): JSX.Element => {
  return (
    <Layout title='Blogs' description='Blogs'>
      <Grid container>
        <Grid item xs={1} lg={2}></Grid>
        <Grid item xs={10} lg={8}>
          <Categories />
          <BlogsList />
        </Grid>
        <Grid item xs={1} lg={2}></Grid>
      </Grid>
    </Layout>
  );
};
