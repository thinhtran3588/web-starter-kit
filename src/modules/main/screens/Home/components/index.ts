export * from './CoverImage';
export * from './Features';
export * from './Quote';
export * from './Help';
