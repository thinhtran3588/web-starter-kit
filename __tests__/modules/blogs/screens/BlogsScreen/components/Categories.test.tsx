import React from 'react';
import { render } from '@testing-library/react';
import { Categories } from '@app/modules/blogs/screens/Blogs/components';

describe('@app/modules/blogs/screens/Blogs/components/Categories', () => {
  beforeEach(() => {});

  it('renders successfully', async () => {
    const { baseElement } = render(<Categories />);
    expect(baseElement).toMatchSnapshot();
  });
});
