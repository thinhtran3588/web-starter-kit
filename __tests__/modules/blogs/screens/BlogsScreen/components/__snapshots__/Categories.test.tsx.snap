// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@app/modules/blogs/screens/Blogs/components/Categories renders successfully 1`] = `
<body>
  <div>
    <div
      class="makeStyles-categoriesContainer-1"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-6"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-2 MuiPaper-rounded"
              >
                <div>
                  <h2
                    class="makeStyles-categoriesTitle-3"
                  >
                    // This impressive paella
                  </h2>
                  <p
                    class="makeStyles-categoriesDescription-4"
                  >
                    Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui
                  </p>
                </div>
                <div
                  class="makeStyles-categoriesButtonContainer-5"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      Explore
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card2-6 MuiPaper-rounded"
              >
                <div>
                  <h2
                    class="makeStyles-categoriesTitle-3"
                  >
                    // This impressive paella
                  </h2>
                  <p
                    class="makeStyles-categoriesDescription-4"
                  >
                    Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui
                  </p>
                </div>
                <div
                  class="makeStyles-categoriesButtonContainer-5"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      Explore
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card3-7 MuiPaper-rounded"
          >
            <div>
              <h2
                class="makeStyles-categoriesTitle-3"
              >
                // This impressive paella
              </h2>
              <p
                class="makeStyles-categoriesDescription-4"
              >
                Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui
              </p>
            </div>
            <div
              class="makeStyles-categoriesButtonContainer-5"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Explore
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card4-8 MuiPaper-rounded"
          >
            <div>
              <h2
                class="makeStyles-categoriesTitle-3"
              >
                // This impressive paella
              </h2>
              <p
                class="makeStyles-categoriesDescription-4"
              >
                Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui
              </p>
            </div>
            <div
              class="makeStyles-categoriesButtonContainer-5"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Explore
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
